### DISCUSSION TITLE
How the access orbitControls (from warpSpeed)

### DISCUSSION THREAD

**Discussion by namlit4** [enable3d/enable3d#164] [Help / Questions]
i am struggling with getting access to orbitControls. I tried 

`const { orbitControls } = this.warpSpeed('-ground')`

but this yields "undefined". Also no success with 

`const { Scene3D, FLAT, THREE, OrbitControls } = ENABLE3D`

My goal is to alter properties of orbitControls, llike damping etc. Any solution to this?

**Comment by yandeu** [comment]
`warpSpeed()` returns a promise.

Use `const { orbitControls } = await this.warpSpeed('-ground')`

Or chain a `.then()`.

*There are some examples that show how it works. I'm sure you will find one 🙂*
